### PR TITLE
GAWB-1161 -- UI portion

### PR DESCRIPTION
**Pull request by skwalker**
This goes along with https://github.com/broadinstitute/agora/pull/170, so to test this with that agora PR, you can change the remoteMethodsUrl and remoteConfigurationsUrl in MethodsService.scala in Orchestration so that they're set to your local agora ones (for me it was https://172.17.0.4/api/v1/methods or /configurations).

- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Make sure Swagger is updated if API changes
- [x] **Submitter**: If updating admin endpoints, also update [firecloud-admin-cli](https://github.com/broadinstitute/firecloud-admin-cli)
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] Tell the tech lead (TL) that the PR exists if they wants to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [x] **TL** sign off
- [x] **LR** sign off
- [x] **Assign to submitter** to finalize
- [x] **Submitter**: Verify all tests go green, including CI tests
- [x] **Submitter**: Squash commits and merge to develop
- [x] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [ ] **Submitter**: Inform other teams of any API changes via hipchat and/or email
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/cljs/org/broadinstitute/firecloud_ui/common/components.cljs [hunk 1]
```diff
@@ -273,6 +273,7 @@
       [:div {:style {:float "left"}}
        (make-field entity "createDate" "Created: " false common/format-date)
        (make-field entity "entityType" "Entity Type: " false)
+       (make-field entity "managers" "Owners: " false (partial clojure.string/join ", "))
```
Comment (davidangb) [inline src/cljs/org/broadinstitute/firecloud_ui/common/components.cljs:276]: needs to be "owners", not "managers".

Comment (rushtong) [reply src/cljs/org/broadinstitute/firecloud_ui/common/components.cljs:276]: @davidangb - The "Owners" values that come back from orchestration is actually in a json field with a key of `managers`

**Comment by rushtong** [comment]
👍 LG, WG

File: src/cljs/org/broadinstitute/firecloud_ui/common/components.cljs (hunk 1 above)
Comment (skwalker) [reply src/cljs/org/broadinstitute/firecloud_ui/common/components.cljs:276]: yeah sorry David -- I changed the json agora is returning to managers

Comment (davidangb) [reply src/cljs/org/broadinstitute/firecloud_ui/common/components.cljs:276]: my bad - must have been an error in my environment. I see "managers" in the json response now, all good!

**Comment by davidangb** [comment]
👍

**Review by davidangb — APPROVED** [review]

**Pull request merged**